### DISCUSSION TITLE
Console / Switch support?

### DISCUSSION THREAD

**Discussion by nightblade9** [HaxeFlixel/flixel#2418] [Q&A] [answered]
Hi,

Back on the old forums, there used to be a post (by Lars Doucet, I think) mentioning that HaxeFlixel (via Lime) had a path to run on consoles. That was a few years ago and it looks like that link is dead.

Does HaxeFlixel running on consoles? I'm specifically interested in the Switch, but good to know for any other consoles that might be supported.

**Comment by Geokureli** [comment, marked as answer]
https://www.fortressofdoors.com/setting-boundaries-open-source-gamedev-on-consoles/
here's the most recent article by Lars regarding console support. the skinny: Switch - yes. Others - no. Maintaining openfl/flixel for switch is a tough job for one man, especially when Nintendo spits out patches at a stupid pace, so it may be behind, here and there.

**Comment by larsiusprime** [comment]
People are still shipping stuff with the "official" switch backend I
sporadically maintain, others are sunsetted. There *do* exist some people
out there with their own private (typically commercial) backends out there
if you know where to look -- Ratalaika is a particularly popular port house
for this sort of thing, but there's probably others.

If you need access to the Switch port you need to be approved by Nintendo
first, and then it's right in their middleware page to request access; that
sends me an email from them telling me to give you the stuff.

On Wed, Sep 29, 2021 at 11:25 AM Geokureli ***@***.***> wrote:

>
> https://www.fortressofdoors.com/setting-boundaries-open-source-gamedev-on-consoles/
> here's the most article by Lars regarding console support. the skinny:
> Switch - yes. Others - no. Maintaining openfl/flixel for switch is a tough
> job for one man, especially when Nintendo spits out patches at a stupid
> pace, so it may be behind, here and there.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/HaxeFlixel/flixel/discussions/2418#discussioncomment-1403972>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFUN6FS4CSPIFF6XWBFID3UEM4XZANCNFSM5FAGOC7Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
www.fortressofdoors.com -- My design & business of game development blog

**Comment by nightblade9** [comment]
Ah yes, I remember reading this in 2019. Thank you so much!

**Comment by Slushi-Github** [comment]
Silly question, I've been researching for a while about using OpenFL and Lime (hence HaxeFlixel) on consoles, specifically Nintendo consoles, but, I've come to the same result, that I need to be inside the Nintendo Developer program and stuff, and that if it's not for developing on the Nintendo Switch, I mean if I wanted to do it on the Wii U, I would have to use older versions of OpenFL if I wanted to. 

But, I've been interested in developing for Wii U with HaxeFlixel, but, it's impossible nowadays right? Nintendo no longer allows to develop for Wii U or 3DS, and I'm not in the Nintendo Developer program, I guess for legal issues I can't get the Lime dependencies to compile to this console, right?
